src/Post/Features/SettingsPage.php

```diff
@@ -7,6 +7,8 @@
 
 namespace OomphInc\Bases\Post\Features;
 
+use OomphInc\Bases\Admin\SettingsPage as Page;
```

Comment:
This is a matter of style but I would always import the class using the same name as the source class, so that this would be `... as SettingsPage`, instead. Just for consistency.